resources/views/event_list.blade.php

```diff
@@ -0,0 +1,126 @@
+<!doctype html>
```

Comment:
I think you should extend from your master file. Review our discussion about the Blade Directives, or read here from the documentation: https://laravel.com/docs/5.8/blade#extending-a-layout

Reply:
I checked the document.
I used Blade Templates.